qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py

```diff
@@ -18,8 +18,14 @@
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column('collection', sa.Column('description', sa.String(), nullable=True))
-    op.add_column('collection', sa.Column('group', sa.String(length=100), nullable=True))
-    op.excecute("UPDATE collection SET group=default")
+
+    op.add_column('collection', sa.Column('group', sa.String(), nullable=False, server_default='default'))
```

Comment:
what does `op.alter_column('collection', 'group', server_default=None)` do?

Reply:
For some reasons strings do not work here, it thinks we are trying to set a composite column (similar to a C struct). So we set the column with a server default meaning that any `None` will be replaced by that value (default on creation) and then we turn off the default forcing pydantic to provide it. 

Similar operation to the `UPDATE ... SET ...` calls.